starter-code/iteration-1/main.js

```diff
@@ -0,0 +1,20 @@
+let arr1 = [];
+let addTask = task => {
+    arr1.push(task);
+};
```

Comment:
En consola os sale: 
"Number of items: undefined"
Habia que añadir "return arr1.length"

Lo mismo en las otras iteraciones y en "deleteTask"...
Ya, ya sé que en los comentarios del código que habeis bajado no ponía eso...